packages/api-sdk/package.json

```diff
@@ -0,0 +1,140 @@
+{
+  "name": "@recallnet/api-sdk",
+  "version": "0.0.4",
+  "author": "Speakeasy",
+  "type": "module",
+  "bin": {
+    "mcp": "bin/mcp-server.js"
+  },
+  "tshy": {
+    "sourceDialects": [
+      "@recallnet/api-sdk/source"
+    ],
+    "exports": {
+      ".": "./src/index.ts",
+      "./package.json": "./package.json",
+      "./types": "./src/types/index.ts",
+      "./models/errors": "./src/models/errors/index.ts",
+      "./models/components": "./src/models/components/index.ts",
+      "./models/operations": "./src/models/operations/index.ts",
+      "./*.js": "./src/*.ts",
+      "./*": "./src/*.ts"
+    }
+  },
+  "sideEffects": false,
+  "scripts": {
+    "lint": "eslint --cache --max-warnings=0 src",
+    "build:mcp": "bun src/mcp-server/build.mts",
+    "build": "npm run build:mcp && tshy",
```

Comment:
same question here re: `npm` usage...is that a speakeasy thing, or should we be using `pnpm` here?

Reply:
This is generated by Speakeasy